LayoutTests/fast/events/constructors/keyboard-event-constructor.html

```diff
@@ -80,8 +80,8 @@
 shouldBeEqualToString("new KeyboardEvent('eventType', { keyIdentifier: {koakuma: 12345} }).keyIdentifier", "[object Object]");
 shouldBeEqualToString("new KeyboardEvent('eventType', { keyIdentifier: {valueOf: function () { return 'koakuma'; } } }).keyIdentifier", "[object Object]");
 
-// location / keyLocation is passed.
-["location", "keyLocation"].forEach(function (propertyName) {
+// location is passed.
+["location"].forEach(function (propertyName) {
```

Comment:
Entirely optional, but there's no need for a `forEach` construct here anymore since it's just a single property.

<details>
<summary>Reduce complexity in the test case</summary>

```diff
diff --git a/LayoutTests/fast/events/constructors/keyboard-event-constructor.html b/LayoutTests/fast/events/constructors/keyboard-event-constructor.html
index 0b381c4db3e0..09761747e7a2 100644
--- a/LayoutTests/fast/events/constructors/keyboard-event-constructor.html
+++ b/LayoutTests/fast/events/constructors/keyboard-event-constructor.html
@@ -80,40 +80,36 @@ shouldBeEqualToString("new KeyboardEvent('eventType', { keyIdentifier: [1, 2, 3]
 shouldBeEqualToString("new KeyboardEvent('eventType', { keyIdentifier: {koakuma: 12345} }).keyIdentifier", "[object Object]");
 shouldBeEqualToString("new KeyboardEvent('eventType', { keyIdentifier: {valueOf: function () { return 'koakuma'; } } }).keyIdentifier", "[object Object]");

-// location / keyLocation is passed.
-["location", "keyLocation"].forEach(function (propertyName) {
-    // numbers within the unsigned long range.
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": KeyboardEvent.DOM_KEY_LOCATION_STANDARD })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": KeyboardEvent.DOM_KEY_LOCATION_LEFT })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_LEFT");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": 4294967294 })." + propertyName, "4294967294");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": 4294967295 })." + propertyName, "4294967295");
-
-    // numbers out of the unsigned long range.
-    // 2^{53}-1, the largest number that can be exactly represented by double.
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": 9007199254740991 })." + propertyName, "4294967295");
-    // 2^{64}-1
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": 18446744073709551615 })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": 12345678901234567890 })." + propertyName, "3944679424");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": -1 })." + propertyName, "4294967295");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": 123.45 })." + propertyName, "123");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": NaN })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-
-    // Non-numeric values.
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": undefined })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": null })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": '' })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": '12345' })." + propertyName, "12345");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": '' })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": '12345' })." + propertyName, "12345");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": '12345a' })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": 'abc' })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": [] })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": [12345] })." + propertyName, "12345");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": [12345, 67890] })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": {} })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": {moemoe: 12345} })." + propertyName, "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
-    shouldBe("new KeyboardEvent('eventType', { " + propertyName + ": {valueOf: function () { return 12345; }} })." + propertyName, "12345");
-});
-shouldBe("new KeyboardEvent('eventType', { keyLocation: KeyboardEvent.DOM_KEY_LOCATION_LEFT }).location", "KeyboardEvent.DOM_KEY_LOCATION_LEFT");
-shouldBe("new KeyboardEvent('eventType', { location: KeyboardEvent.DOM_KEY_LOCATION_LEFT }).keyLocation", "KeyboardEvent.DOM_KEY_LOCATION_LEFT");
+// location is passed.
+// numbers within the unsigned long range.
+shouldBe("new KeyboardEvent('eventType', { location : KeyboardEvent.DOM_KEY_LOCATION_STANDARD }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : KeyboardEvent.DOM_KEY_LOCATION_LEFT }).location", "KeyboardEvent.DOM_KEY_LOCATION_LEFT");
+shouldBe("new KeyboardEvent('eventType', { location : 4294967294 }).location", "4294967294");
+shouldBe("new KeyboardEvent('eventType', { location : 4294967295 }).location", "4294967295");
+
+// numbers out of the unsigned long range.
+// 2^{53}-1, the largest number that can be exactly represented by double.
+shouldBe("new KeyboardEvent('eventType', { location : 9007199254740991 }).location", "4294967295");
+// 2^{64}-1
+shouldBe("new KeyboardEvent('eventType', { location : 18446744073709551615 }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : 12345678901234567890 }).location", "3944679424");
+shouldBe("new KeyboardEvent('eventType', { location : -1 }).location", "4294967295");
+shouldBe("new KeyboardEvent('eventType', { location : 123.45 }).location", "123");
+shouldBe("new KeyboardEvent('eventType', { location : NaN }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+
+// Non-numeric values.
+shouldBe("new KeyboardEvent('eventType', { location : undefined }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : null }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : '' }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : '12345' }).location", "12345");
+shouldBe("new KeyboardEvent('eventType', { location : '12345a' }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : 'abc' }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : [] }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : [12345] }).location", "12345");
+shouldBe("new KeyboardEvent('eventType', { location : [12345, 67890] }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : {} }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : {moemoe: 12345} }).location", "KeyboardEvent.DOM_KEY_LOCATION_STANDARD");
+shouldBe("new KeyboardEvent('eventType', { location : {valueOf: function () { return 12345; }} }).location", "12345");
```

</details>

Reply:
Thanks. Done in [3c3bc33](https://github.com/WebKit/WebKit/commit/3c3bc33da04b37943b123aaf42fa138c8389ffe5).